MetaMask/metamask-mobile
# chore: use dweb.link as default IPFS gateway

## **Description**

Change default IPFS gateway from `gateway.pinata.cloud` to `dweb.link`.

## **Related issues**

- Targeting: #11985
- #11692
  - https://github.com/MetaMask/mobile-planning/issues/1961
- #11401

## **Manual testing steps**

- Set up a new installation from scratch
- Confirm IPFS requests are succeeding and all IPFS requests going to `dweb.link`

## **Screenshots/Recordings**

### **Before**

### **After**

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
